src/parameterizations/stochastic/MOM_stochastics.F90

```diff
@@ -34,7 +34,7 @@ module MOM_stochastics
 !> This control structure holds parameters for the MOM_stochastics module
 type, public:: stochastic_CS
   logical :: do_sppt         !< If true, stochastically perturb the diabatic
```

Comment:
Need to revise the description of `do_sppt` as well, as per Bob's review.

Reply:
This line is already in main MOM6, and I didn't write it. I think the rest of the comment could read 'parameterization tendencies' but I'm not really sure. Does that need to be dealt with as part of this PR?